components/world.tsx

```
@@ -0,0 +1,204 @@
+
+      const editor = graphEditorRef.current;
+      const world = worldRef.current;
+      const graph = graphRef.current;
+      const editorChanged = editor?.draw() ?? false;
+
+      if (!world || !graph) {
+        return;
+      }
+
+      const revision =
+        typeof (graph as any).getRevision === "function"
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)